eks_fargate/README.md

```diff
@@ -266,6 +315,10 @@ Datadog Agent v6.19+ supports live containers in the EKS Fargate integration. Li
 
 Datadog Agent v6.19+ supports live processes in the EKS Fargate integration. Live processes appear on the [Processes][20] page. To enable live processes, [enable shareProcessNamespace in the pod spec][21].
 
+### Kubernetes resources view
+
+Collecting Kubernetes requires a [Cluster Agent setup][#running-the-cluster-agent-cluster-checks-runner]
```

Comment:
Updated the H4 above so that the dash doesn't bork the anchor link.

```suggestion
To collect Kubernetes resource views, you need a [Cluster Agent setup][#running-the-cluster-agent-or-the-cluster-checks-runner].
```